runtime/vm/jvminit.c

```diff
@@ -8121,6 +8121,28 @@ setSignalOptions(J9JavaVM *vm, J9PortLibrary *portLibrary)
 		}
 	}
 
+#if defined(WIN32) && defined(_WIN32_WINNT_WINBLUE) && (_WIN32_WINNT_MAXVER >= _WIN32_WINNT_WINBLUE)
+	HMODULE h_kernel32 = GetModuleHandle(TEXT("kernel32.dll"));
+
+	if ((NULL != h_kernel32) && IsWindows8OrGreater()) {
+		typedef BOOL (WINAPI * PMP_func)(HANDLE, PROCESS_MITIGATION_POLICY, PVOID, SIZE_T);
```

Comment:
nit: no space after `*`. Then please rebase again due to #21228.

Reply:
> nit: no space after *.

Sorry just double checking to make sure I get it right: the current format is `(WINAPI * PMP_func)(HANDLE,...)`. Is the suggestion to change it to `(WINAPI *PMP_func)(HANDLE,...)` or `(WINAPI * PMP_func) (HANDLE,...)`?  Or basically remove the space after `*PMP_func` or add a space before ` (HANDLE,...)`? 

Here are the existing examples of `typedef` in jvminit.c:
https://github.com/eclipse-openj9/openj9/blob/e3d8d3916daad8240be455829097fd95dc417b1f/runtime/vm/jvminit.c#L139-L140

Reply:
I'm asking for the former (`(WINAPI *PMP_func)(HANDLE, ...)`).

There are many examples of misplaced spaces besides those on lines 139 and 140. If you want to remove the unwanted spaces there, that would be fine.
```c
typedef void (JNICALL *J9_EXIT_HANDLER_PROC)(jint);
typedef void (JNICALL *J9_ABORT_HANDLER_PROC)(void);
```
The motivation for changes such as this is to avoid misleading human readers of the code (`*` binds more tightly to the name that follows as compared with the type that precedes it).

And yes, I realize that I included a space in my earlier suggestion - we all make mistakes.